drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageReportView.java

```diff
@@ -16,11 +16,13 @@
 
 package org.drools.workbench.screens.scenariosimulation.client.rightpanel;
 
+import com.google.gwt.dom.client.ButtonElement;
```

Comment:
Considering that this panel uses elemental2 elements, please use `HTMLButtonElement` instead of GWT one

Reply:
@danielezonca done!